### DISCUSSION TITLE
Mod versions don't show up

### DISCUSSION THREAD

**Discussion by Apollo0000** [KSP-CKAN/CKAN#3627] [Q&A] [answered]
Hello,
I'm trying to download RO & RP-1 to the 1.12.3 version of KSP and Ckan won't work correctly
When ever I open it and look for the mods it doesn't show most of them and the ones it does show are always compatible to very old versions like 1.3.1
Can anyone shed some light on why that is?
![ckan](https://user-images.githubusercontent.com/111250207/184554739-85d397f8-ecee-48f5-a420-2075eca5935f.png)

**Comment by HebaruSan** [comment, marked as answer]
I believe this was caused by a bug that was fixed after this question was asked. The latest CKAN should not have this problem:

- https://github.com/KSP-CKAN/CKAN/releases/latest
